src/ui-strings.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { addStrings } from '@wordpress/components';
+import { select } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+export const uiStrings = () => {
+	const { getSettings } = select( 'core/block-editor' );
+	const unsupportedBlockEditor = {
```

Comment:
I think just using 2 variables instead of grouping them in an object is just simpler and easier to understand, especially since you're assigning an `unsupportedBlockEditor` to a `supported` property